pkg/trace/config/apply.go

```diff
@@ -280,6 +280,9 @@ func (c *AgentConfig) applyDatadogConfig() error {
 	if config.Datadog.IsSet("apm_config.connection_reset_interval") {
 		c.ConnectionResetInterval = getDuration(config.Datadog.GetInt("apm_config.connection_reset_interval"))
 	}
+	if config.Datadog.IsSet("apm_config.sync_flushing") {
+		c.SynchronousFlushing = config.Datadog.GetBool("apm_config.sync_flushing")
```

Comment:
You should add this as know to this function: https://github.com/DataDog/datadog-agent/blob/90d4d955067ca0e43764aa339112f007bb4836b3/pkg/config/apm.go. And also add an env. var for it?

It might also have to be in the config template along with proper documentation: https://github.com/DataDog/datadog-agent/blob/90d4d955067ca0e43764aa339112f007bb4836b3/pkg/config/config_template.yaml. But I'll let you decide.

Reply:
I think I'll add it to setKnown, but I don't think I'll add it to the template because user's of the regular trace agent shouldn't toggle it.

Reply:
Goti it.